interfaces/builtin/cpu_control.go

```diff
@@ -82,6 +82,11 @@ const cpuControlConnectedPlugAppArmor = `
 
 # Allow control CPU C-states switching see: https://docs.kernel.org/power/pm_qos_interface.html#pm-qos-framework
 /dev/cpu_dma_latency rw,
+
+# Allow interrupt affinity settings
```

Comment:
could you also add a link to the kernel docs? https://www.kernel.org/doc/html/latest/core-api/irq/irq-affinity.html

Reply:
Off course, sure!